### ISSUE TITLE
Give feedback about the status of the robot

### ISSUE THREAD

**Issue by mklomp** [mirte-robot/mirte-web-interface#54]
Currently no feedbakc whatsoever is given to the user. It would be helpful to get info about the status of telemetrix, ROS, and the websockets. The UI could be grayed out as long as there is no connection yet.

**Comment by knmcguire** [comment]
Yes this was exactly my thought. Would make it much easier to debug
